app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -35,19 +35,14 @@ def secrets(host)
           {
             'per-host' => {
               host.name => {
-                'ansible_ssh_pass' => rex_ssh_password(host),
-                'ansible_sudo_pass' => rex_sudo_password(host)
+                'ansible_become_password' => rex_effective_user_password(host)
               }
             }
           }
         end
 
-        def rex_ssh_password(host)
-          host_setting(host, 'remote_execution_ssh_password')
-        end
```

Comment:
Do we really want to do this? The ssh password can still be different from the effective user one

Reply:
Yeah, I should only change sudo_pass. Got confused in ansible_ssh_pass and ansible_su_pass. :P 
But I think we should change ansible variable from "ansible_sudo_pass" to "ansible_become_password", it will help to deal with both sudo and su user method.
I will add the ansible_ssh_pass again.